### DISCUSSION TITLE
0.4.1

### DISCUSSION THREAD

**Discussion by kkinder** [kkinder/puepy#55] [Announcements]
## What's Changed
* Update README.md by @kishankarun in https://github.com/kkinder/puepy/pull/43 and https://github.com/kkinder/puepy/pull/53
* Update watchers.py by @kishankarun in https://github.com/kkinder/puepy/pull/47
* Use threading http server by @kkinder in https://github.com/kkinder/puepy/pull/54

## New Contributors
* @kishankarun made their first contribution in https://github.com/kkinder/puepy/pull/43

**Full Changelog**: https://github.com/kkinder/puepy/compare/release/0.4.0...release/0.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kkinder/puepy/releases/tag/release/0.4.1'>0.4.1</a>.</em>

**Comment by kishankarun** [comment]
Glad to be the contributor.

Thanks for accepting the contributions 🙏
